composer.json

```diff
@@ -3,21 +3,32 @@
 	"description": "Provides email notifications for visitor comments",
 	"type": "silverstripe-module",
 	"license": "BSD-3-Clause",
-	"authors": [
-		{
-			"name": "Damian Mooyman",
-			"email": "damian@silverstripe.com"
-		}
-	],
+	"authors": [{
+		"name": "Damian Mooyman",
+		"email": "damian@silverstripe.com"
+	}],
 	"require": {
-		"silverstripe/comments": "^2.0"
+		"silverstripe/comments": "^3.0"
 	},
 	"require-dev": {
-		"phpunit/PHPUnit": "~3.7@stable"
-	},
+        "phpunit/PHPUnit": "^5.7",
+        "squizlabs/php_codesniffer": "^3.0"
+    },
+    "scripts": {
+        "lint": "phpcs src/ tests/",
+        "fix": "php-cs-fixer fix src/ && php-cs-fixer fix tests/"
```

Comment:
So this came from comments which probably originally came from one of the core modules at some point ago.

We haven't been including these scripts in the composer config recently, but if we're going to they should use the binaries included with code sniffer in the composer config, e.g. `vendor/bin/phpcs` and `vendor/bin/phpcbf` rather than assuming they'd be globally installed as well

Thoughts?